contracts/mocks/token/ERC20uRWAMock.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {uRWA20} from "../../token/ERC20/extensions/ERC20uRWA.sol";
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+
+// solhint-disable-next-line contract-name-capwords
+abstract contract uRWA20Mock is uRWA20, AccessControl {
```

Comment:
```suggestion
contract uRWA20Mock is uRWA20, AccessControl {
```
?
Do we need the mock to be abstract?

Reply:
It's not needed but it helps because hardhat-exposed exposes the constructors automatically 😄